guides/source/maintenance_policy.md

```diff
@@ -59,6 +59,11 @@ be built from 1.2.2, and then added to the end of 1-2-stable. This means that
 security releases are easy to upgrade to if you're running the latest version
 of Rails.
 
+Only direct security patches will be included in security releases. Fixes to address
+non-security-related bugs or other implications of a security patch may be published
```

Comment:
```suggestion
non-security related bugs related to a security patch may be published
```

Reply:
Thanks for the speedy review!

Just to clarify the thought behind "other implications", I would generally understand "bug" to mean "unintended", whereas my impression (which may be wrong) is that the changes in #38063 to make the `TestSession` behaviour consistent were _knowingly_ left out of the original security patch due not being directly necessary to fix the security issue at hand.

Reply:
I assumed this was related to https://github.com/rails/rails/commit/892eab777c418135ce0646e91bc9ebb08a29ab9b not being included in the security release for https://github.com/rails/rails/commit/2a52a38cb51b65d71cf91fc960777213cf96f962. This, to me, is an unintended bug introduced by the security patch because the security patch was made without considering Ruby 2.2 requirements.

https://github.com/rails/rails/pull/38063 appears to be a side effect of the security patch too, which is knowingly left out for the same reason the syntax error fix was. I think it is implied that subsequent changes wouldn't be included in another release of 5.2.x. Without the full context, I think the document is more concise not mentioning other implications.

Reply:
On reflection, I think the point is still made sufficiently without the "other implications". I'll amend the commit with the suggested changes now.